app/src/androidTest/java/com/google/samples/apps/nowinandroid/ui/NavigationTest.kt

```diff
@@ -251,11 +263,14 @@ class NavigationTest {
     }
 
     @Test
-    fun navigationBar_multipleBackStackInterests() {
+    fun navigationBar_multipleBackStackInterests() = runTest {
         composeTestRule.apply {
             onNodeWithText(interests).performClick()
-            // TODO: Grab string from fake data
-            onNodeWithText("Android Studio & Tools").performClick()
+
+            // Select a random topic
+            val topic = datasource.getTopics().random().name
```

Comment:
It's not good practice to use anything random in tests as it can make them flaky. Select the last element instead.